pkg/tuf/verify.go

```diff
@@ -10,58 +10,44 @@ import (
 	log "github.com/sirupsen/logrus"
 	"github.com/theupdateframework/notary/client"
 
+	"github.com/cnabio/cnab-go/bundle"
 	"github.com/cnabio/signy/pkg/cnab"
 )
 
-// VerifyCNABTrust ensures the trust metadata for a given GUN matches the metadata of the pushed bundle
-func VerifyCNABTrust(ref, trustServer, tlscacert, trustDir, timeout string) error {
-	target, trustedSHA, err := GetTargetAndSHA(ref, trustServer, tlscacert, trustDir, timeout)
-	if err != nil {
-		return err
-	}
-	log.Infof("Pulled trust data for %v, with role %v - SHA256: %v", ref, target.Role, trustedSHA)
-
-	log.Infof("Pulling bundle from registry: %v", ref)
-	bun, err := cnab.Pull(ref)
-	if err != nil {
-		return fmt.Errorf("cannot pull bundle: %v", err)
-	}
-	buf, err := json.MarshalCanonical(bun)
-	if err != nil {
-		return err
-	}
-
-	err = verifyTargetSHAFromBytes(target, buf)
-	if err == nil {
-		log.Infof("The SHA sums are equal: %v\n", trustedSHA)
-	}
-
-	return err
-}
+// VerifyTrust ensures the trust metadata for a given GUN matches the metadata of the pushed bundle
+func VerifyTrust(ref, localFile, trustServer, tlscacert, trustDir, timeout string) (*client.TargetWithRole, []byte, error) {
+	var bun *bundle.Bundle
+	var buf []byte
 
-// VerifyFileTrust ensures the trust metadata for a given GUN matches the computed metadata of the local file
-func VerifyFileTrust(ref, localFile, trustServer, tlscacert, trustDir, timeout string) error {
```

Comment:
The reason these two methods were initially split was because I assumed at some point we will have slightly different processes for verifying a thin vs. thick bundles.